src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMapping.java

```diff
@@ -49,6 +55,25 @@ public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData) {
         }
     }
 
+    public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData, ReferenceDataServiceHolder referenceDataServiceHolder) {
+        log.info("Get the next hearing channel {}", referenceDataServiceHolder.isAdjournmentFlagEnabled());
+        if (referenceDataServiceHolder.isAdjournmentFlagEnabled() && nonNull(caseData.getAdjournment().getTypeOfNextHearing())) {
```

Comment:
There should be 4 tests for this:
true && null
true && non null
false && null
false && non null

Reply:
Added the new test case to cover all the scenarios.

Reply:
Coverage is 100% now